app/models/metadatum.rb

```diff
@@ -114,9 +123,17 @@ class Metadatum < ApplicationRecord
     extraction_batch: "Extraction Batch",
     sample_unit: "Sample Unit",
     life_stage: "Life Stage",
-    id_method: "ID Method",
-    genus_species: "Genus/Species",
-    blood_fed: "Blood Fed"
+    preservation_method: "Preservation Method",
+    trap_type: "Trap Type",
+    blood_fed: "Blood Fed",
+    reported_sex: "Reported Sex",
+    comp_sex: "Comp. ID Sex",
```

Comment:
I do not think it makes sense to shorten words on the description.

Reply:
"Comp. ID Sex" does look weird. I'll do "Computed Sex". The full description is "Computationally Identified Sex", but that seems a little long for a label.